packages/langium/src/test/langium-test.ts

```diff
@@ -34,20 +35,13 @@ export function parseHelper<T extends AstNode = AstNode>(services: LangiumServic
 export type ExpectFunction = (actual: unknown, expected: unknown, message?: string) => void;
 
 let expectedFunction: ExpectFunction = (actual, expected, message) => {
-    if (typeof expect === 'function') {
-        if (message && expect.length === 2) {
-            // With `jest-expect-message`
-            expect(actual, message).toEqual(expected);
-        } else {
-            expect(actual).toEqual(expected);
-        }
-    } else {
-        throw new Error('No expect function provided. Use the `expectFunction` function to supply a custom expect function or install `vitest` or `jest`.');
-    }
+    assert.deepStrictEqual(actual, expected, message);
 };
 
 /**
- * Overrides the assertion function used by tests. Tries to use Jest by default.
+ * Overrides the assertion function used by tests. Uses `assert.deepStrictEqual` by default
+ *
+ * @deprecated Since 1.2.0. Do not override the assertion functionality.
```

Comment:
Why not keeping this (non-deprecated)? Would leave us flexibility!

Reply:
I'm not sure how much this flexibility is actually an advantage. `assert.deepStrictEqual` should already cover all of our bases, especially since it's only used by the test functions that we export.